mongodbatlasv2/api_organizationsatlas_stores.go

```diff
@@ -0,0 +1,56 @@
+ package latest
+
+import (
+	"context"
+	openapiclient "github.com/mongodb/go-client-mongodb-atlas"
+)
+
+type OrganizationsApiStore struct {
```

Comment:
That is just example of 44 different stores that were created automatically from client